docs/release-notes/dictionary-releases.md

```diff
@@ -20,6 +20,31 @@ None to report.
 
 ------>
 
+## Release 1.13
+
+**Release Date: November 17, 2021**
+
+### Updates
+
+- The "AJCC 6th Edition" has been added to the controlled terminology for tumour staging systems. This will allow data submitters to submit TNM categories and stage groups for the AJCC 6th Edition.
+- Optional clinical fields related to dose intensity reduction have been added to the Chemotherapy table. These include `dose_intensity_reduction`, `dose_intensity_reduction_event` and `dose_intensity_reduction_amount`.
+- Optional clinical fields related to laterality have been added. These include the `laterality` field in the Primary Diagnosis table and the `specimen_laterality` field in the Specimen table.
```

Comment:
should we say "file" instead of "table"? does the audience know this as a "table"?

Reply:
@kcullion Rosi was using "table" in the past Release notes, so I thought maybe to stay consistent to keep using "table".

Reply:
Ok let's keep it as table then! thanks :)